x-pack/legacy/plugins/rollup/public/crud_app/store/middleware/clone_job.js

```diff
@@ -4,19 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { getRouter, getUserHasLeftApp } from '../../services';
```

Comment:
I thought that this was needed because of angular's `$https`, but I was wrong. Moreover removing `getUserHasLeftApp` produced the issue: after rollup job creation you could not navigation to other plugins, the app just redirected to the job list.
Thank you a lot for pointing me out. I reverted this deletion.